CMPUT301Project/app/src/main/java/com/example/GroupProject/User.java

```diff
@@ -5,21 +5,22 @@
 
 package com.example.GroupProject;
 
+import android.app.Application;
+
 /**
  * User class
  * Class of a user, containing username, first name and last name
  * @Author Kyle Bricker
  *
  */
 public class User {
-    String username;
-    String firstName;
-    String lastName;
 
-    User(String username, String firstName, String lastName) {
+    private String username;
+    private String firstName;
+    private String lastName;
+
+    User(String username) {
         this.username = username;
```

Comment:
Initialize first and lastname here. Add back the other constructor for future use.